environment/src/main/java/com/sequenceiq/environment/experience/liftie/ExperiencesByLiftie.java

```diff
@@ -39,18 +40,18 @@ public ExperiencesByLiftie(ExperienceIndependentLiftieClusterWorkloadProvider wo
     @Override
     public int getConnectedClusterCountForEnvironment(EnvironmentExperienceDto environment) {
         throwIfTrue(environment == null, () -> new IllegalArgumentException(EnvironmentExperienceDto.class.getSimpleName() + " cannot be null!"));
-        List<ClusterView> clusterViews = getClusterViewsForAllProvidedWorkload(environment.getName(), environment.getAccountId());
+        List<ClusterView> clusterViews = getClusterViewsForWorkloads(environment.getName(), environment.getAccountId());
         return countNotDeletedClusters(clusterViews);
     }
 
     @Override
     public void deleteConnectedExperiences(@NotNull EnvironmentExperienceDto environment) {
         throwIfTrue(environment == null, () -> new IllegalArgumentException(EnvironmentExperienceDto.class.getSimpleName() + " cannot be null!"));
         LOGGER.debug("Getting Liftie cluster list for environment '{}'", environment.getName());
-        List<ClusterView> clusterViews = getClusterViewsForAllProvidedWorkload(environment.getName(), environment.getAccountId());
+        List<ClusterView> clusterViews = getClusterViewsForWorkloads(environment.getName(), environment.getAccountId());
         LOGGER.debug("Starting Liftie clusters deletion for environment '{}'", environment.getName());
         clusterViews.stream()
-                .filter(cv -> LiftieIgnorableStatuses.notContains(cv.getClusterStatus().getStatus()))
+                .filter(cluster -> LiftieIgnorableClusterStatuses.notContains(cluster.getClusterStatus().getStatus()))
                 .forEach(cv -> liftieApi.deleteCluster(cv.getClusterId()));
```

Comment:
cv -> cluster to be consistent with the previous line

Reply:
done